docs/overview/community/ComponentGuidelines.stories.mdx

```diff
@@ -0,0 +1,178 @@
+import { Title, Meta } from "@storybook/addon-docs";
+
+<Meta title="Overview/Community/Component Guidelines" />
+
+<Title>Component Guidelines</Title>
+
+This guide aims to provide insights and recommendations for building and contributing
+robust and adaptable components that are consistent with the existing UI Kit library.
+
+## File Structure & Purpose
+
+The directory should contain the following files:
+
+```sh
+MyComp/
+├── index.ts            # entry file with the component's public exports
+├── MyComp.tsx          # component implementation and types
+├── MyComp.stories.tsx  # component documentation and examples
+├── MyComp.styles.ts    # component classes and stying utils
+└── MyComp.test.tsx     # component unit tests
+```
+
+## Guidelines
+
+- Components and their types MUST be named in PascalCase and prefixed with `Hv`.
+- Components MUST be exported from the `index.ts` file.
```

Comment:
"Components and their types" MUST be exported from the `index.ts` file. => Just to make sure they don't forget the `MyCompProps` and `MyCompPropsClasses`.